.gitignore

```
@@ -5,3 +5,5 @@ build/
 __pycache__/
 .tox
 .idea
```

Not strictly needed, but useful to add in all repos
```suggestion
.idea
venv/
```